### ISSUE TITLE
Date estimate of next Castle level

### ISSUE THREAD

**Issue by ghost** [mainIine/foe-helfer-extension#2232]
It would be great to be able to see a rough estimate, when the next level of the Castle system will be reached. 
Since there are many moving parts, I think a prediction based on preferred optimal play would be sufficient, e.g. the user can select "Daily", "Challenge", etc., and then the maximum of those point values are used for the calculation of the estimate. 
The output would then read "x days until.." or "level up on dd/mm/yyyy"

**Comment by outoftheline** [comment]
i dont think we will consider implementing this, because the higher levels take ages
